queries.yaml

```diff
@@ -273,3 +273,28 @@ post_process:
     params:
       source_layer: roads
       start_zoom: 9
+  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 14
+      properties: [is_bridge, is_tunnel, oneway]
+      where: >-
+        (kind == 'path' and zoom < 15) or
+        (kind in ['minor_road', 'major_road', 'highway', 'rail'] and zoom < 13)
```

Comment:
Generally other web maps only show `oneway` starting at zoom 15 (because symbols showing the one way need to fit between street grid intervals). But `is_bridge`, `is_tunnel` is used currently starting at zoom 13 (to change the outline road casing to sort above the road inline). Should the be split apart (similar to highway and `name`, `ref` below)?